code/modules/shuttle/custom_shuttle.dm

```diff
@@ -8,6 +11,8 @@
 	req_access = list( )
 	possible_destinations = "whiteship_home"
 
+	var/obj/item/shuttle_creator/designator = null
```

Comment:
This could potentially cause a hard delete if the shuttle creator is deleted, due to this still referencing the shuttle creator upon shuttle creator deletion. If this isn't accessed *too* frequently, you could use a weakref or in the destroy() proc of shuttle_creator find the attached comtroller and dereference it.

Reply:
I changed it to a weakref. I tested it and it seems to be fine, but this is the first time I've used a weakref so I'm not sure.